symfony/asset-mapper/6.4/assets/app.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Welcome to your app's main JavaScript file!
+ *
+ * This file will be included onto the page via the importmap() Twig function,
+ * which should already be in your base.html.twig.
+ */
+console.log('This log comes from assets/app.js - welcome to AssetMapper! 🎉')
```

Comment:
We’re also going to import ./styles/app.as from inside this file now. And then NOT add a link tag for app.css in base.html.twig

Reply:
OK, I've made those changes now.